fiorella/src/styles/layout/_footer.scss

```diff
@@ -0,0 +1,92 @@
+footer {
+  padding: 10px 15px;
+
+  .box-footer {
+    border: 2px solid $cerise-red;
+    text-align: center;
+    width: 100%;
+
+    @media #{ $tablet }, #{ $desktop } {
+      display: flex;
+      justify-content: space-around;
+      padding-bottom: 15px;
+    }
+  }
+}
+
+.hide-menu {
+  display: none;
+
+  @media #{ $tablet }, #{ $desktop } {
+      display: block;
+  }
+}
+
+.title-footer {
+  color: $cerise-red;
+  font-family: $playfair-display;
+}
+
+.ul-item-menu {
+  margin: 0;
+  padding: 0;
+
+  li {
+    list-style: none;
+    margin: 15px 0;
+
+    a{
+      color: $text-color-black;
+      font-family: $merriweather-sans;
+      font-weight: $font-light;
+      text-decoration: none;
+
+      &:hover {
+        color: $cerise-red;
+        font-weight: bold;
+      }
+    }
+  }
+
+  .social-media {
+    display: flex;
+    justify-content: space-around;
+    padding-bottom: 15px;
+
+    li{
+      a {
+        .icon {
```

Comment:
Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 5